great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -100,6 +100,12 @@ def name(self) -> str:
     def batch_request(self) -> Optional[Union[BatchRequest, RuntimeBatchRequest, dict]]:
         return self._batch_request
 
+    @batch_request.setter
+    def batch_request(
+        self, batch_request: Union[BatchRequest, RuntimeBatchRequest, dict]
+    ) -> None:
+        self._batch_request = batch_request
+
```

Comment:
@cdkini Same thought -- please see the reconciliation tests.

Reply:
Done!